src/AdminSettings.vue

```diff
@@ -110,6 +111,25 @@
 				</p>
 			</li>
 		</ul>
+		<ul class="password-generator__settings-list">
+			<li>
+				<CheckboxRadioSwitch :checked.sync="config.genPhrases"
+					type="switch"
+					@update:checked="updateBoolSetting('genPhrases')">
+					{{ t('password_policy', 'Generate passpharses') }}
+				</CheckboxRadioSwitch>
+			</li>
+			<li>
```

Comment:
Will be good use anything like v-if at this place and only display the word count field if genPhrases is checked.
```suggestion
			<li v-if="config.genPhrases">
```